tests/vmi_controller_test.go

```diff
@@ -0,0 +1,36 @@
+package tests_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"kubevirt.io/kubevirt/tests/decorators"
+	"kubevirt.io/kubevirt/tests/libdomain"
+	"kubevirt.io/kubevirt/tests/libvmifact"
+	"kubevirt.io/kubevirt/tests/libvmops"
+)
+
+var _ = Describe("[sig-compute]Controller devices", decorators.SigCompute, func() {
```

Comment:
Can you break this out into a separate commit with a clear description of why this is required within this PR please.